shopelectro/management/commands/test_db.py

```diff
@@ -118,11 +121,20 @@ def get_name(number, parent=None):
     def create_products(self, categories, tags):
         def create_images(page: Page):
             def create_image(file_path, slug):
-                Image.objects.create(
+                # product "/catalog/products/2/" contains image
+                image = Image.objects.create(
                     model=page,
                     slug=slug,
                     image=ImageFile(open(file_path, mode='rb'))
```

Comment:
We are opening here the same file as below and does not release it. We should group these lines by one file's context:
```python
with open(file_path, mode='rb') as file_:
    image = Image.objects.create(
        ...,
        image=ImageFile(file_)
    )
    ...
    shutil.copyfile(file_.name, file_dst_path)
```

Reply:
Done.